src/Extensions.cs

```diff
@@ -717,13 +717,17 @@ public static string GetConstraint(string name, string constraintName, string co
                 // swagger spec states that InclusiveMaximum should be a number
                 // however the validation code supports int64 and float64.  to be
                 // on the safe side handle both cases here.
-                if (constraintValue.IndexOf('.') > -1)
+                switch (type)
                 {
-                    value = $"float64({constraintValue})";
-                }
-                else
-                {
-                    value = $"int64({constraintValue})";
+                    case "float64":
+                        value = $"float64({constraintValue})";
+                        break;
+                    case "int32":

```

Comment:
If you're adding `int32` you might as well add `float32` above.

Reply:
Well, in [this file](https://github.com/Azure/autorest.go/blob/cd02b097dc9a7ba3d77cdbc466882c7ceb2bc91c/src/Model/PrimaryTypeGo.cs#L60) we do not support `float32` in the SDK, therefore I suppose maybe we could also omit this one?

Reply:
Good point, we can omit it here.